README.md

```diff
@@ -1 +1,178 @@
-# Ruby-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Test](#test)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog-of-Things <a name="about-project"></a>
+
+**Catalog of Things** is a ruby written console application which delivers a detailed method to create different objects of a catalog such as item, books, label, music album, genre, game, author information. The primary purpose of the program is to explore the object oriented programming. Here we have demonstrated the creation of objects, adding the basic UI, unit test the methods and preserve the data in json files.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>None</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>None</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **DRY method**
+- **Catalog of different things like: Book, Music and Game**
+- **Add Storage feature**
+- **Unit Test**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/Kingno3l/Ruby-Capstone.git
+
+### Install
+
+```sh
+  cd catalog
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following steps:
+
+```sh
+  ruby main.rb
+```
+
+### Test
+
+To run the unit test files, run the following command in the terminal:
+
+```sh
+  rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Video
+
+<!-- LIVE DEMO -->
+
+LIVE DEMO
+
+> Watch a [live](https://drive.google.com/file/d/18wpZbqrEjaDEDqOtd6Weq1cla9Er1z75/view?usp=sharing) demo of the project.
+
```

Comment:
- - Video requirement:
![Selection_061](https://github.com/Kingno3l/Ruby-Capstone/assets/107632780/2cb70ae0-ec94-4e83-9c45-eb07eedaadff)

- [ ] You have done well creating the presentation video, but while going through the video, I found out you left out one of the requirements. According to the project requirement, each team member needs to present and Highlight `one interesting piece of code or something you built that you are very proud of`. I suggest you incorporate this into your video. :+1: :100: